### PR TITLE
pytorch checkpoint files added to gitignore

### PR DESCRIPTION
**Pull request by run-me**
It would be great to have pytorch checkpoints files added to .gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,6 +25,9 @@ wheels/
 *.egg
 MANIFEST
 
+# Pytorch
+*.pth
```
Comment (hellock) [inline .gitignore]: Thanks! We can add custom rules at the bottom, which may be easier to check.

Comment (run-me) [reply .gitignore]: Thanks for the fast review, made the changes and follow the same for the future PR. Cheers!

Comment (karlind) [reply .gitignore]: .pth is saved to `work_dirs` by default, which has already been ignored.

Comment (run-me) [reply .gitignore]: adding .pth gives user the flexibility to keep checkpoints any where in the project and also we dont need to explicitly specify or request them to keep checkpoints in specific directory. thanks!

**Comment by openmmlab-v** [comment]
Hi @run-me ！First of all, we want to express our gratitude for your significant PR in the mmdetection project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/raweFPmdzG

If you have WeChat，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Pull request merged**